### DISCUSSION TITLE
Support for passing certificate and private key as the actual raw string & I/O stream object

### DISCUSSION THREAD

**Discussion by mohan314e** [projectdiscovery/httpx#1249] [Ideas]
### Please describe your feature request:

Please add features to support to pass certificate and private key as the actual raw string as well I/O stream object to 'cert' param which is we use for Client Side Certificates.

### Describe the use case of this feature:

for now as 'cert' param only takes file path for that reason we have to store the file in server then we can pass the path. but if this features is added then we can directly pass raw string which can be stored in DataBase or also if it support I/O stream object then we can store somewhere remote and read and pass the I/O object. this will increase the flexibility. it is now completely locked to store the file in server only.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
